ospo-mindmap/Content/ospomindmap.md

```diff
@@ -109,24 +189,87 @@ incubation and post launch
   - Vet, organize and operate OS projects
   - Train and coach OS Leaders
 
-### OS Leadership
+#### Optimize Open Outbound Source Contributions
+
+
+### 🤝 Collaborate with Open Source Organizations
+
+##### 📖 Open Standards
+
+- OpenChain
+- SPDX
+
+##### 🏠 Foundations
+
+- Apache Foundation
+- Free Software Foundation Europe
+  - Legal Network
+- Eclipse Foundation
+- Linux Foundation
+  - FINOS
+  - OpenSSF
+  - OpenChain
+  - SPDX
+  - TODO Group
+- Open Source Initiative
+- OW2
+  - Good Governance Initiative
 
-#### Provides open source consultancy and become a strategic partner
 
-- Evaluate the behavior
-and performance of the project
-- Advise on which open source technologies
-to adopt or remove
-- Provide insights into OSS trends
+### ⏱️ Track Performance Metrics
+
+- Ability to assess health of open source projects
+- Measure community engagement
+- Development of open source leaders
+
+### 🤝 Implement InnerSource Practices
+
+- Increase productivity, quality, code re-use, transparency and trust
+- Establish communication channels across the organization
+- Improve collaboration across the organization
+- Remove organizational silos and identify talent across the organization
+
+
+### 🫶 Grow and Retain Open Source Talent Inside the Organization
+
+- Train new and coach existing open source leaders
+- Drive an internal open source ambassadors program
+- Recognize contributions of organization's members
+
+
+### 🧑‍💼 Open Source Adviser
```

Comment:
"Open Source Adviser" sounds more like a role rather than a responsibility. Maybe we can change it to "Give advice on the Open Source Ecosystem"

Reply:
Changed the section title to 'Give Advice on Open Source' and added 'Open Source Advisor' to roles. Dropped the Ecosystem bit as per section 4.9 of @ibrahimhaddad's [OSPOs PDF](https://github.com/ibrahimhaddad/publications/blob/master/Open%20Source%20Program%20Offices.pdf), advise is for both internal and external and I presume most people will confuse 'Ecosystem' with Open Source community.